DEPLOY.md

```diff
@@ -1,9 +1,10 @@
-# Deployment
+## Deployment
 
-ebmbot is deployed on smallweb1 and is managed by systemd.
+Deployment uses `dokku` and requires the environment variables defined in `dotenv-sample`.
```

Comment:
This implies (to me) that it reads the environment variables from `dotenv-sample`, but I don't think that's the case.  Can you clarify?

Reply:
No, this is a leftover from when I thought I was going to put dummy env vars in dotenv-sample, but eventually didn't. It's only there for local development now